eng/common/templates/steps/source-build.yml

```diff
@@ -63,6 +63,11 @@ steps:
       targetRidArgs='/p:TargetRid=${{ parameters.platform.targetRID }}'
```

Comment:
this changes eng/common so we need to make sure it is updated in dotnet/arcade or it'll get overwritten with the next arcade update

Reply:
I've created https://github.com/dotnet/arcade/pull/10782.